uat/features/stanford_image.feature

```diff
@@ -1,9 +1,10 @@
+@stanford
 Feature: Stanford Image
   In order to ensure that upgrades do not break existing functionality
   As an administrative user
   I want to ensure that the Stanford Image module is working properly
 
-  @api @javascript
+  @api @javascript @safe @deploy
```

Comment:
Same comment here as for context_list_active. Not sure if we should enable this as part of the install profile. The self-service install profile needs some love.